### PR TITLE
doc: misc. updates

### PR DESCRIPTION
**Pull request by jorgeorpinel**
- Dvcignore (and check-ignore) related updates per https://github.com/iterative/dvc.org/pull/1629#pullrequestreview-463015446 et al.
- get/import -o description update
- more remote add/modify updates per [this chat](https://discord.com/channels/485586884165107732/485596304961962003/741357885052813313)

### CODE REVIEW

File: content/docs/command-reference/import.md [hunk 1]
```diff
@@ -75,11 +75,10 @@ from the source repo.
 
 ## Options
 
-- `-o <path>`, `--out <path>` - specify a path (directory and/or file name) to
-  the desired location to place the imported file in (instead of using the
-  current working directory). If an existing directory is specified, the target
-  data will be placed inside. If `path` contains a parent directory which does
-  not exist, this command will fail.
+- `-o <path>`, `--out <path>` - specify a path to the desired location to place
+  the imported file in the workspace (instead of using the current working
```
Comment (shcheklein) [inline content/docs/command-reference/import.md:79]: can it be imported directory?

File: content/docs/command-reference/get.md [hunk 2]
```diff
@@ -55,11 +55,9 @@ name.
 
 ## Options
 
-- `-o <path>`, `--out <path>` - specify a path (directory and/or file name) to
-  the desired location to place the download file in. The default value (when
-  this option isn't used) is the current working directory (`.`) and original
-  file name. If an existing directory is specified, then the target data will be
-  placed inside.
+- `-o <path>`, `--out <path>` - specify a path to the desired location to place
+  the imported file in the workspace (instead of using the current working
```
Comment (shcheklein) [inline content/docs/command-reference/get.md:59]: imported?

File: content/docs/command-reference/import.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/import.md]: Yes. Updated.

File: content/docs/command-reference/get.md (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/get.md]: Oops. "downloaded". Fixed.

**Pull request merged**